deploy/local/generate_win_solution.py

```diff
@@ -140,9 +140,14 @@ def generate_proj(config, relative_example_path):
 
 def find_all_examples(examples_dir):
     examples = []
-    with cd(examples_dir):
-        for relative_path in glob.iglob('**/*.cpp', recursive=True):
-            examples.append(relative_path)
+
+    print('Walking dir {}'.format(examples_dir))
+    for root, dirnames, filenames in os.walk(examples_dir):
+        print('-> checking file: {}{}'.format(root,filenames))
```

Comment:
Is this print really necessary or it was added for debugging? We already have prints in  `generate_sln` and `write_proj`.

Reply:
I will remove it. Let me complete testing